src/app/tests/suites/certification/Test_TC_ACT_3_2.yaml

```diff
@@ -33,7 +33,7 @@ tests:
           "Step 0: Preparation: TH as server exposes an Actions server cluster
           on EP 1, with one action (supporting all possible commands) and
           corresponding ActionLists and EndpointLists attributes: ActionList:
-          contains one list element (ActionListStruct):  ActionIO = 0x1001  Name
+          contains one list element (ActionListStruct):  ActionIO = 0xA001  Name
```

Comment:
Can we just update test plan to use 0x1001 instead of  0xA001 , so that we don't need to update script and reference app.

Reply:
script is reverted back to use action 0x1001

Reply:
Seems like the revert itself got reverted due to a bad merge?